### PR TITLE
Repo cleanup/update

### PR DESCRIPTION
**Pull request by shtirlic**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+obj
```
Comment (vitaliy-aksyonov) [inline .gitignore:1]: These masks may be too broad. Consider to use more specific paths.

**Review by vitaliy-aksyonov — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+/obj
+bld*.inc
```
Comment (vitaliy-aksyonov) [inline .gitignore:2]: What is this file?

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+build
```
Comment (vitaliy-aksyonov) [inline .gitignore:3]: Some IDEs create another directories for build. Something like "build-Release", etc.

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+mygolded.h
```
Comment (vitaliy-aksyonov) [inline .gitignore:4]: /golded3/mygolded.h

**Review by vitaliy-aksyonov — CHANGES_REQUESTED** [review]
Maybe just remove .travis.yml and leave that .gitignore alone?
I have local gitignore for my own configuration and it works just fine.

File: .gitignore (hunk 2 above)
Comment (shtirlic) [reply .gitignore:2]: try run make in repo

Comment (shtirlic) [reply .gitignore:2]: image

File: .gitignore (hunk 3 above)
Comment (shtirlic) [reply .gitignore:3]: this is not IDE specific

**Review by vitaliy-aksyonov — APPROVED** [review]

File: .gitignore (hunk 2 above)
Comment (shtirlic) [reply .gitignore:2]: also my make failed with
```
/usr/bin/ld: ../lib/lnx/libhunspell.a(suggmgr.o): in function `SuggestMgr::SuggestMgr(char const*, int, AffixMgr*)':
/home/shtirlic/projects/golded-plus/goldlib/hunspell/suggmgr.cxx:38:(.text+0x62): undefined reference to `AffixMgr::get_encoding()'
/usr/bin/ld: /home/shtirlic/projects/golded-plus/goldlib/hunspell/suggmgr.cxx:39:(.text+0x6d): undefined reference to `get_current_cs(char const
*)'
```

File: .gitignore (hunk 4 above)
Comment (shtirlic) [reply .gitignore]: fixed

**Review by vitaliy-aksyonov — APPROVED** [review]

**Pull request merged**